code/game/machinery/teleporter.dm

```diff
@@ -585,6 +621,7 @@
 /obj/machinery/teleport/station/screwdriver_act(mob/user, obj/item/I)
 	if(default_deconstruction_screwdriver(user, "controller-o", "controller", I))
 		update_icon(UPDATE_ICON_STATE)
+		update_icon(UPDATE_OVERLAYS)
```

Comment:
```suggestion
		update_icon(UPDATE_ICON_STATE | UPDATE_OVERLAYS)
```
Be sure to go over the rest as well. `update_icon` expects a bitflag value now. So all the values here are bitflags, which you can combine using ` | `